### PR TITLE
Logs improvement

### PR DESCRIPTION
**Pull request by shoaibhr**
separator for logs each time script re-runs and add line number of command associated with log, and rotate log file if the size exceeds 5MB, keep 3 backup

### CODE REVIEW

File: script_runner/main.py [hunk 1]
```diff
@@ -1,27 +1,19 @@
...
-concurrently using threads.
-
-Logging is implemented to track the success or failure
-of each command, providing insights into the script's operations.
-
-Usage:
-1. Specify the path to the .env file and the configuration file containing commands.
-2. The script loads the environment variables and executes the commands in parallel.
-3. Logs are generated for monitoring the execution process.
-"""
```
Comment (mhamidjamil) [inline script_runner/main.py:15]: Please add back these comments @shoaibhr have you run pylint on this file?

File: script_runner/main.py [hunk 2]
```diff
@@ -33,54 +25,63 @@ def load_env_file(env_file_path: str):
...
+                    commands.append((line_number, None, parts[0]))
     except FileNotFoundError as e:
-        logging.error("Configuration file not found: %s", file_path)
-        logging.exception(e)
-    except Exception as e:  # pylint: disable=W0718
-        logging.error("An error occurred while reading the configuration file.")
-        logging.exception(e)
+        logger.error("Configuration file not found: %s", file_path)
+        logger.exception(e)
+    except Exception as e:
```
Comment (mhamidjamil) [inline script_runner/main.py:51]: pylint will cause issue on this part too, please make sure you use pylint

**Review by mhamidjamil — CHANGES_REQUESTED** [review]
Please run pylint and let me know when you are done with pylint issues

**Comment by mhamidjamil** [comment]
@shoaibhr I have updated pylint git action please check pylint issues and ping when you are done with them

File: script_runner/main.py (hunk 1 above)
Comment (shoaibhr) [reply script_runner/main.py:15]: added

**Review by mhamidjamil — APPROVED** [review]
LGTM

**Pull request merged**